### DISCUSSION TITLE
Postal not using IP from the pool

### DISCUSSION THREAD

**Discussion by moazam1** [postalserver/postal#1875] [Help with using Postal]
Hi, I have enabled IP pool for a domain. I have setup hostname: smtp.domain.com and IP: 178.xx.xx.xxx and assigned to a client. But whenever I send email it uses the default IP. I am sending email through smtp.domain.com not using default.domain.com

I am also having issue sending emails with TLS. I am not sure how can I use SSL certificate? for smtp.domain.com I am using vesta CP to proxy domain to postal server and it's also taking care of SSL as well. Can someone guide me how can I fix TLS issue?

**Comment by willpower232** [comment]
the only thing I can suggest for the IP Pool is to verify that the IP address appears on the server when you do `ip a` on the terminal

Regarding the certificate, you will need to find the file your control panel is using and add that into the config folder for Postal then update the config file. Alternatively, you could source a separate certificate for the same domain name.

**Reply by moazam1** [reply]
Hi @willpower232 I am behind router and using local IPs with NAT. When I use ip a I do see 2 IPs. I am able to ping IP and also I am able to open smtp.domain.com which opens spark page. I am not sure what else I have to do?

Noticed a similar thread here: https://github.com/postalserver/postal/issues/323 but unfortunately no solution posted. Any help will be greatly appreciated.

**Reply by willpower232** [reply]
so the IP you're pinging is the routers IP? because local IPs aren't public?

**Reply by moazam1** [reply]
@willpower232 no I am pinging public IPs and they are working fine. I am also able to ping smpt.domain.com which I assigned pool IP. I am even using smtp.domain.com as a smtp host address but still postal is not picking up IP from pool.

**Reply by reetikpant** [reply]
@willpower232 i'm also using the hostname correctly, where can i find the ip4 and ip6 address to add, i dont know what my ip4 and ip6??
